docs/source/transforms.rst

```diff
@@ -33,6 +35,11 @@ images of a given batch, but they will produce different transformations
 across calls. For reproducible transformations across calls, you may use
 :ref:`functional transforms <functional_transforms>`.
 
+The following examples illustate the use of the available transforms:
+
+    * :ref:`sphx_glr_auto_examples_plot_transforms.py`
+    * :ref:`sphx_glr_auto_examples_plot_scripted_tensor_transforms.py`
```

Comment:
Thanks! Just wondering, do you know if it's possible to have the card with the illustrations for the tutorials?

Not merge blocking, just for my curiosity

Reply:
The `.. minigallery` directive doesn't seem to work well with modules, but we can add the rendered plots manually:

![image](https://user-images.githubusercontent.com/1190450/117666761-1e855080-b19c-11eb-8f3a-51a99f150501.png)

Thanks for the suggestion I think it looks nice!